src/transform.js

```diff
@@ -155,7 +200,8 @@ function transformSourceCodeString(sourceCode, addSetup=true) {
   const output = babel.transformSync(sourceCode, {
     plugins: pluginArray,
   });
-  return output.code;
+
+  return {code: output.code, title: transformInfo.description};
```

Comment:
For this kind of non-trivial return values, it's best to have some documentation.

Reply:
How should this documentation be done? Should I just expand on the comment above the function that mentions the return object?

Reply:
Yup. You could also use a bit of JSDoc here.